includes/discovery/sensors/dbm/bti800.inc.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+echo 'BTI800 dBm';
+
+$multiplier = 1;
+$divisor = 1;
+
+foreach ($pre_cache['bti800'] as $index => $entry) {
+    if ($entry['sfpInfoWigth'] != '0') {
+        $oidRx = '.1.3.6.1.4.1.30005.1.7.100.1.2.6.3.1.8.' . $index;
+        $oidTx = '.1.3.6.1.4.1.30005.1.7.100.1.2.6.3.1.7.' . $index;
+        $currentRx = snmp_get($device, $oidRx, '-Oqv', 'BTI8xx-SFP-MIB', '/opt/librenms/mibs/bti');
+        $currentTx = snmp_get($device, $oidTx, '-Oqv', 'BTI8xx-SFP-MIB', '/opt/librenms/mibs/bti');
```

Comment:
- the `mibdir` option should be `bti`
- you should use one of the snmp "multi" functions to make one call instead of multiple to the device

Reply:
Ok noted @Jellyfrog. I'm able to change it instead using snmp_get() for each return value.